presto-docs/src/main/sphinx/connector/localfile.rst

```diff
@@ -0,0 +1,45 @@
+====================
+Local File Connector
+====================
+
+The local file connector allows querying data stored on the local
+filesystem of each worker.
+
+Configuration
+-------------
+
+To configure the local file connector, create a catalog properties file
+``etc/catalog/localfile.properties`` with the following contents:
+
+.. code-block:: none
+
+    connector.name=localfile
+
+Using the Local File Connector
+------------------------------
+List the available system schemas::
+
+    SHOW SCHEMAS FROM system;
+
+
+Local File Connector Schemas and Tables
+---------------------------------------
+The local file connector provides a single schema ``logs`` that contains tables for logs.
+You can see all the available tables by running ``SHOW TABLES``::
+
+        SHOW TABLES FROM localfile.logs;
+
+
+``http_request_log``
+^^^^^^^^^^^^^^^^^^^^
+This table contains the http request logs from each node on the cluster.
+
+Configuration Properties
+------------------------
+
+================================================== ============================================================
+Property Name                                      Description
+================================================== ============================================================
+
+    ``presto-logs.http-request-log-location``       Directory or file where http request logs are written
+    ``presto-log.http-request-log-pattern``         If http-request-log-location is a directory, file name regex for files in this directory
```

Comment:
Also, is it really a regex? If so, please update the connector to use globbing, as that is standard for file matching:

https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob